graphein/ppi/graphs.py

```diff
@@ -0,0 +1,150 @@
+# %%
+# Graphein
+# Author: Arian Jamasb <arian@jamasb.io>, Ramon Vinas
+# License: MIT
+# Project Website: https://github.com/a-r-j/graphein
+# Code Repository: https://github.com/a-r-j/graphein
+import logging
+from typing import Callable, List, Optional
+
+import networkx as nx
+
+from graphein.ppi.config import PPIGraphConfig
+from graphein.utils import (
+    annotate_edge_metadata,
+    annotate_graph_metadata,
+    annotate_node_metadata,
+    compute_edges,
+)
+
+log = logging.getLogger(__name__)
+
+
+EDGE_COLOR_MAPPING = {"string": "r", "biogrid": "b"}
+
+
+def parse_kwargs_from_config(config: PPIGraphConfig) -> PPIGraphConfig:
+    """
+    If configs for STRING and BIOGRID are provided in the Global PPIGraphConfig, we update the kwargs
+    :param config:
```

Comment:
Missing docstring. Here's a suggestion:

```suggestion
    :param config: PPI graph configuration object.
```